docs/source/faq/installation.md

```diff
@@ -0,0 +1,172 @@
+# FAQ: Installation
+
+(different-versions)=
+## Why are there different versions of Manim?
+
+Manim was originally created by Grant Sanderson as a personal project and for use
+in his YouTube channel,
+[3Blue1Brown](https://www.youtube.com/channel/UCYO_jab_esuFRV4b17AJtAw).
+As his channel gained popularity, many grew to like the style of his animations and
+wanted to use manim for their own projects. However, as manim was only intended for
+personal use, it was very difficult for other users to install and use it.
+
+In late 2019, Grant started working on faster OpenGL rendering in a new branch,
+known as the `shaders` branch. In mid-2020, a group of developers forked it into what is
+now the community edition; this is the version documented on this website.
+In early 2021, Grant merged the shaders branch back into master, making it the default branch in his repository -- and this is what `manimgl` is.
+The old version, before merging the `shaders` branch is sometimes referred to as
+`ManimCairo` and is, at this point, only useful for one singular purpose: rendering
+Grant's old videos locally on your machine. It is still available in his GitHub
+repository in form of the `cairo-backend` branch.
+
+To summarize:
+- [**Manim**, or **ManimCE**](https://manim.community) refers to the community
+  maintained version of the library. This is the version documented on this website;
+  the package name on PyPI is [`manim`](https://pypi.org/project/manim/).
+- [ManimGL](https://github.com/3b1b/manim) is the latest released version of the
+  version of the library developed by Grant "3b1b" Sanderson. It has more experimental
+  features and breaking changes between versions are not documented. Check out
+  its documentation [here](https://3b1b.github.io/manim/index.html); on PyPI the
+  package name is [`manimgl`](https://pypi.org/project/manimgl/).
+- [ManimCairo](https://github.com/3b1b/manim/tree/cairo-backend) is the name that
+  is sometimes used for the old, pre-OpenGL version of `manimgl`. It is not available
+  via PyPI.
```

Comment:
If I'm right, on PyPI it's called `manimlib`.

Reply:
It is manimgl: https://pypi.org/project/manimgl/

Reply:
`manimgl` is the new Opengl version right? and the old Cairo version is still available as `manimlib`: https://pypi.org/project/manimlib/, afaik previously the version from 3b1b repo was published to `manimlib` package.

Reply:
Ah, I misread the line you were commenting -- now I understand! Last release in February 2022 sounds right too.

```suggestion
- [ManimCairo](https://github.com/3b1b/manim/tree/cairo-backend) is the name that
  is sometimes used for the old, pre-OpenGL version of `manimgl`. It is available
  on [PyPI as `manimlib`](https://pypi.org/project/manimlib/),
  but to compile old projects you likely have to
  install the correct version from source regardless.
```